File: pset1/operations/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Size: ");
    }
    while(n < 1);

    for(int i = 0; i < n; i++)
    {
        printf("?");
    }
    printf("\n");
}

File: pset6/exercises/mario.py
from cs50 import get_int

def height():
    while True:
        n = get_int("Height: ")
        if n > 0 and n <= 8:
            break
    return n

def pyramid(n):
    hash = n - (n - 1)
    dot = n - 1
    for i in range(n):
        print(" " * dot + "#" * hash)
        hash += 1
        dot -= 1

height = height()
pyramid(height)